src/evrythng/entities/application_users.py

```diff
@@ -74,22 +78,33 @@ def authenticate_facebook_user(expires, token, api_key=None):
 
 
 def delete_user(user_id, api_key=None):
-    """Delete a User"""
+    """Delete a User."""
     assertions.datatype_str('user_id', user_id)
     url = '/users/{}'.format(user_id)
     return utils.request('DELETE', url, api_key=api_key)
 
 
 def logout_user(api_key=None):
+    """Logout an Application User."""
     return utils.request('POST', '/auth/all/logout', api_key=api_key)
 
 
+def login_user(email, password, api_key):
```

Comment:
All other api_key arguments in all other functions have a default of None... is there a reason this one doesn't?